### PR TITLE
Use Scan.addColumn(...) instead  of Scan.addFamily(...) when has qualifier.

### PR DESCRIPTION
**Pull request by q977734161**
when columns contain family and column name,using Scan.addColumn(...) method to reduce network pressure and decrease unnecessary columns.

### CODE REVIEW

File: hbase/src/main/java/org/apache/metamodel/hbase/HBaseDataContext.java [hunk 1]
```diff
@@ -222,9 +222,11 @@ protected DataSet materializeMainSchemaTable(Table table, List<Column> columns,
         for (Column column : columns) {
             if (!column.isPrimaryKey()) {
                 final int colonIndex = column.getName().indexOf(':');
+                final int len = column.getName().length();
                 if (colonIndex != -1) {
                     String family = column.getName().substring(0, colonIndex);
-                    scan.addFamily(family.getBytes());
+                    String colName = column.getName().substring(colonIndex + 1, len);
```
Comment (kaspersorensen) [inline hbase/src/main/java/org/apache/metamodel/hbase/HBaseDataContext.java]: Remember to add `final` :-)

File: hbase/src/main/java/org/apache/metamodel/hbase/HBaseDataContext.java [hunk 2]
```diff
@@ -222,9 +222,11 @@ protected DataSet materializeMainSchemaTable(Table table, List<Column> columns,
...
+                    scan.addColumn(family.getBytes(),colName.getBytes());
```
Comment (kaspersorensen) [inline hbase/src/main/java/org/apache/metamodel/hbase/HBaseDataContext.java]: ... and a space after the comma :-)

Comment (q977734161) [reply hbase/src/main/java/org/apache/metamodel/hbase/HBaseDataContext.java]: Ok,I will modify the code at once and thanks Peter kaspersorensen for reviewing.

**Comment by kaspersorensen** [comment]
Sorry, seems the test flakyness is not completely over. Can you commit another empty commit to trigger rebuild?

**Comment by q977734161** [comment]
ok,I will commit another empty commit to trigger rebuild.

**Comment by kaspersorensen** [comment]
Merging this and #219

**Pull request closed without merging**